### PR TITLE
Update changelog for 5.7.0

### PR DESCRIPTION
**Pull request by will-moore**
Based on https://github.com/ome/omero-py/milestone/4?closed=1

(A couple of trivial PRs not mentioned).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# 5.7.0 (June 2020)
+
+- Fix: pywin import in Python 3.7 ([#203](https://github.com/ome/omero-py/pull/203))
+- Support use of CLI within OMERO.scripts ([#186](https://github.com/ome/omero-py/pull/186))
+- Fix: Don't unset ICE_CONFIG on Windows ([#193](https://github.com/ome/omero-py/pull/193))
+- Fix: Ignore owner in omero.data.dir check ([#208](https://github.com/ome/omero-py/pull/208))
+- Fix: JSON parsing on Python 3.5 ([#213](https://github.com/ome/omero-py/pull/213))
+- Add new conn.chownObjects() method ([#195](https://github.com/ome/omero-py/pull/195))
```
Comment (sbesson) [inline CHANGELOG.md:8]: I like the usage of `Fix:` above, can we use a similar semantics to communicate new backward-compatible APIs e.g. `API: ` or even sub-sections?

Comment (will-moore) [reply CHANGELOG.md]: 9d4ff5e uses `API` and `Info`. Sound OK? Alternative suggestions?

Comment (sbesson) [reply CHANGELOG.md]: Main issue is with `Info:` as I am unclear what this term is supposed to communicate.  Is it more generally a new `Feature:` (even though it's simply reporting)?

My only other suggestion would be to split e.g. as `## API additions` and `## Bug fixes and other improvements` and remove the prefixes.

Comment (will-moore) [reply CHANGELOG.md]: I guess `Feature` is better.
But I also like the split of API additions and others...

**Comment by will-moore** [comment]
Updated requirement to `omero-py>=5.7.0` in https://github.com/ome/omero-web/pull/148/commits/c2c419865f0afaf84f84c712d2b46c11ccd8c3d4

**Pull request merged**